cromshell

```diff
@@ -714,7 +714,8 @@ function execution-status-count()
   assertCanCommunicateWithServer $2
   turtle
   f=$( makeTemp )
-  curl --connect-timeout $CURL_CONNECT_TIMEOUT --max-time $CURL_MAX_TIMEOUT --compressed -s ${2}/api/workflows/v1/$1/metadata?${CROMWELL_METADATA_PARAMETERS} > $f
+  
+  curl --connect-timeout $CURL_CONNECT_TIMEOUT --max-time $CURL_MAX_TIMEOUT --compressed -s "$2/api/workflows/v1/$1/metadata?includeKey=executionStatus&includeKey=backendStatus&expandSubWorkflows=true" > $f
```

Comment:
Ah.  Now that I understand better, can you pull out the `includeKey=executionStatus&includeKey=backendStatus&expandSubWorkflows=true"` string and make it a new variable called `CROMWELL_SLIM_METADATA_PARAMETERS` defined just below `CROMWELL_METADATA_PARAMETERS`?

Reply:
After you do that, can you also update the call in `slim-metadata`?